css/style.css

```diff
@@ -53,7 +54,54 @@
     margin-left: 30px;
   }
 
-  
+  .nav-link {
+    color: #fff;
+    margin-left: 30px;
+  }
+
+  .mobile-menu {
+    /* display: none; */
+    background: #3C3A39;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    z-index: 1;
+    padding: 30px;
+  }
```

Comment:
- [ ] I noticed that the mobile menu does not disappear after resizing to the desktop width. Kindly fix this issue.
Cheers 👍 

![Habeeb's Portfolio](https://user-images.githubusercontent.com/60618877/179868421-00d35dd3-553c-462d-ab8d-f20560877ddd.gif)